src/util/BlankNodeManager.h

```diff
@@ -236,11 +235,22 @@ class BlankNodeManager {
   // any queries are performed.
   [[nodiscard]] Block allocateExplicitBlock(uint64_t blockIdx);
 
+  // If the `uuid` of the `entry` is not yet registered with this
+  // `BlankNodeManager`, register and return a new `Blocks` struct with the
+  // explicit `uuid`, and explicitly allocate all blocks represented by the
+  // `entry` and store them in the result. If the `uuid` is already registered,
+  // then return a `shared_ptr` to the `Blocks` associated with this `uuid`.
+  // This functionality is used to reinstate sets of registered blocks when
+  // loading SPARQL UPDATEs or serialized cache results when QLever is started.
```

Comment:
UUID is not a variable or typename in this comment, right? It used to, but is no longer. Please update the comment accordingly.